spectator/spectatord.py

```diff
@@ -0,0 +1,35 @@
+import socket
+import threading
+import logging
+
+from spectator.id import MeterId
+
+logger = logging.getLogger("spectator.SpectatorD")
+
+
+class SpectatorD:
+    """Maintain a UDP socket to SpectatorD for publishing Atlas metrics."""
+
+    address = ("127.0.0.1", 1234)
```

Comment:
While it shouldn't change often, I'm not sure we want this hardcoded. There might be times where we want to change it, or maybe someone needs to run a special spectatord process with different settings.

Reply:
A `SidecarConfig` class has been implemented. Went about it a slightly different way - leverage dictionaries instead of functions.